### DISCUSSION TITLE
MEETING: esites Working group - 9 May 2023: 15:00 – 16:30

### DISCUSSION THREAD

**Discussion by rondlg** [fieldmuseum/EMu-Documentation#811] [MEETINGS]
# AGENDA

### [Question 61: (Turks and Caicos) Island details supersede political details in summary strings? KJ - IZ](https://github.com/fieldmuseum/EMu-Documentation/discussions/621)](https://github.com/fieldmuseum/EMu-Documentation/discussions/621)
 - **HOMEWORK:** KGJ and RDP to check status of this discussion

### [Question 50: Is it ok for a site record to exist solely to link two (or more) parents together? IZ - KJ, RDLP, JD](https://github.com/fieldmuseum/EMu-Documentation/discussions/816)](https://github.com/fieldmuseum/EMu-Documentation/discussions/816)
-  

### [Question 51: Is verbatim locality information sufficient justification to create a unique site record? IZ - KJ, RDLP, JD](https://github.com/fieldmuseum/EMu-Documentation/discussions/817)
 - Yes, it’s fine to do that. Rank is Precise Locality.
 - A blank Precise Locality indicates that more information eg lat lon may be found at a later time.

### [Question 55: How best do we deal with the issue of bottom depth with regards to georeferencing marine sites? IZ - KJ, RDLP, JD](https://github.com/fieldmuseum/EMu-Documentation/discussions/829)](https://github.com/fieldmuseum/EMu-Documentation/discussions/829)
- Some places indicate dredge depth as a negative value. What do we want to do? 
- - ANT - Similarity to excavation? Java sea wreck?
- - ZOO/Fishes - depth in Coll.Events
- - IZ: Coll.Event depth = depth of the net or collection method; Site depth = depth/bathymetry of the locality

- dwc terms: 
> https://dwc.tdwg.org/list/#dwc_maximumDistanceAboveSurfaceInMeters
> https://dwc.tdwg.org/list/#dwc_maximumElevationInMeters

- **Immediate solution**: Use Elevation (Relative) for depth below surface. The values here will be negative. Corresponds to [DistanceAboveSurfaceInMeters](https://dwc.tdwg.org/list/#dwc_maximumDistanceAboveSurfaceInMeters). Update Field Level Help.

- **HOMEWORK:** IT/IZ -- IZ has data in CE that will need to be taken and moved into sites.
- - If others have similar data to contact RDP/KGJ and get in on the helpdesk ticket
- Next round development: Consider relabeling 
- Q. Should bottom depth in CE also go in Elevation (relative) in Sites for other areas?
- - A.Yes

- While depth may change, in Sites, the depth/elevation-data is part of the geolocation of a given locality [at a given time]
- Over time, can relate overlapping older/new sites to each other view synonymy-workflows (e.g. [here](https://docs.google.com/document/d/1YA87zYvahTko2McaCWmTofg7XBdjKAbfZjJSG6KfS6Q/edit#heading=h.aux68lr9iqu1))

### [Question 57: Should ‘reef’ be a rank in Class: Aquatic waterbody details? kj-iz](https://github.com/fieldmuseum/EMu-Documentation/discussions/831)](https://github.com/fieldmuseum/EMu-Documentation/discussions/831)
- “Reef” & “Peninsula” can fit in Special Geographic Region [on ‘Site’ tab] (SitGeographicRegion_tab + SitGeographicType_tab)
- “Cape Cod” - 
- - If referring to [cultural] region …
- - If referring to [special/biogeographic region] …
- “Cape” in general?
- - Might not be generalizable; can visit each/case by case in WG
- IPT / regions
- - **HOMEWORK:** Higher Geography mapping & logic - check these

### [Rework esites summary strings to reflect new Classification values](https://github.com/fieldmuseum/EMu-Documentation/issues/722)](https://github.com/fieldmuseum/EMu-Documentation/issues/722)
- Summary string value
- Useful [but currently limited] as a ‘Search-all’ field within + between modules (e.g. in attachment fields)

### [[New term to replace International Waters in continent lookup](https://github.com/fieldmuseum/EMu-Documentation/issues/723)](https://github.com/fieldmuseum/EMu-Documentation/issues/723)
- **DECISION**:
- - Default if we don’t know will become “Marine Waters”
- - Only use “International Waters” when it is explicitly known that it is.
- - https://github.com/fieldmuseum/EMu-Documentation/wiki/admin-ESTABLISHED-STANDARDS#continents-and-oceans
- - Add in Marine Waters Master record to allow for the cases where you do not know if the aquatic locality is in International Waters or not. 
- **DONE**

### [basics training](https://github.com/fieldmuseum/EMu-Documentation/wiki/admin-TRAINING#esites)
- Done
 
### Grenada still needs West Indies in its record
- **DECISION**: Grenada - make sure that “West Indies” is in the LocPolRegion field.

### 1) georeferenceVerificationStatus - where to put it / what values?

- https://dwc.tdwg.org/list/#dwc_georeferenceVerificationStatus 
- Does the data refer to the place or the specimen?
- What confidence do we have in the generation of the co-ordinates?
- - Is that confidence variable or the same across a given project? (If the same, does a static value help?)
- **Interim workflows** for new standards coming online (e.g. similar to ResourceRelationship / Relationship tab + migration workflows)
- **HOMEWORK**: We will need the controlled vocabulary from the project
- - Examples from data currently / RDLP:
- - - unable to georeference
- - - requires georeference
- - - requires verification
- - - verified by data custodian
- - - verified by contributor

- **Short-term** where to put it for us?
- **Longer term** community solution with Axiell

### 2) Status of proposed site record classification recommendation and Summary String rework?
- Need the summarystrings defined before we can change over.

### 3) Status of Waterbody Hierarchy recommendation?
- 

### 4) Record creation help: See comments. Cape Cod; Hillsborough River
- Record creation help: 
- - Cape Cod
- - Hillsborough River
- https://github.com/fieldmuseum/EMu-Documentation/discussions/811#discussioncomment-5841898 
- - [Rivers & River basin workflow steps](https://docs.google.com/document/d/1YA87zYvahTko2McaCWmTofg7XBdjKAbfZjJSG6KfS6Q/edit#heading=h.8si6f6b1n581)

**Comment by kjakymec** [comment]
What's the status of changing site record classification from terrestrial/aquatic to political/geographical?

**Reply by rondlg** [reply]
See above: Needs summary strings defined before transition to new values can occur

**Comment by kjakymec** [comment]
Why do we have to have an ocean at the top of the waterbody hierarchy?

**Reply by rondlg** [reply]
DECISION: See https://github.com/fieldmuseum/EMu-Documentation/issues/723

**Comment by kjakymec** [comment]
Follow-up to question #31 (not in github)on the EMu meeting doc: IZ was asked to provide "the workflows for the data cleaning etc where summary strings are useful. Include the fields that are useful." 1. summary string extremely useful in absence of search-all-fields function; 2. summary sting a useful shortcut for populating fields that link to other modules by isolating a few results via keyword search. John and Robin can speak to this in further detail.

**Comment by kjakymec** [comment]
There are a couple of examples of sites we're not sure how to put in EMu - Cape Code and some sites related to the Hillsborough River in Florida. John can speak to this in greater detail.

**Reply by dangeloje** [reply]
The important thing about the two sites in question is that they are representative of some of the broader issues of entering site records that we have encountered.

Site IRN 732442: In my opinion, the proper Darwin Core representation of this site would have "Point Gammon" in dwc:locality and "North America | United States | Massachusetts | Barnstable | Cape Cod | Yarmouth | West Yarmouth" in dwc:higherGeography. How should this site be entered to achieve this?

Site IRN 731913: Is this site entered correctly? "Hillsborough River" does not show up in the "River" field of Freshwater Details, nor in the summary string of this site record or its freshwater parent.

**Reply by rondlg** [reply]
732442: 
- Suggested solution
![image](https://github.com/fieldmuseum/EMu-Documentation/assets/702709/9f5f3988-df6b-4b57-90a9-c570b5f514df)
- COLLECTIONS HOMEWORK: Will need to make sure this situation is included in the summarydata recommendations topic.
- IT HOMEWORK: include fields in higher geography ipt report mapping.

731913: 
- The summaryData for River is not forming correctly when there is not a river basin as a parent. There is an open ticket with Axiell Hornbill-IN00121052.
- IT HOMEWORK: Poke ticket [DONE]

**Reply by rondlg** [reply]
HORNBILL TICKET IN00121052. See[ Client Testing wiki page](https://github.com/fieldmuseum/EMu-Documentation/wiki/admin-Client-Testing-Ticket-Status#fixed-in-test--version-51-2306301) for status
